onebusaway-android/src/main/res/layout/trip_details_listitem.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (C) 2010-2015 Paul Watts (paulcwatts@gmail.com)
```

Comment:
Should I add my name to files I modified as well, given that you want to keep track of contributors?